CodePush.js

```diff
@@ -8,7 +8,7 @@ import log from "./logging";
 let NativeCodePush = require("react-native").NativeModules.CodePush;
 const PackageMixins = require("./package-mixins")(NativeCodePush);
 
-async function checkForUpdate(deploymentKey = null) {
+async function checkForUpdate(deploymentKey = null, handleBinaryVersionMismatch = null) {
```

Comment:
Please update readme and typings in order to mention about new `handleBinaryVersionMismatch` feature and `checkForUpdate` method signature changes